torch/serialization.py

```diff
@@ -54,6 +54,8 @@
     "LoadEndianness",
     "get_default_load_endianness",
     "set_default_load_endianness",
+    "get_default_mmap_options",
+    "set_default_mmap_options",
```

Comment:
Why was this not here before? Did we silence the public API check on it? :o

Reply:
Don't think we silenced, not sure why it wasn't caught